entrytool/templates/forms/patient_details_form.html

```diff
@@ -1,6 +1,22 @@
 {% load forms %}
 {% load custom_forms %}
-{% select field="PatientDetails.status" change="editing.patient_details.death_cause = null; editing.patient_details.death_date = null" %}
+<div class="form-group">
+  <label class="control-label col-sm-3">
+    Patient Status
+  </label>
+  <div ng-class="{'has-error': form.$submitted && form.status.$error.required }" class="col-sm-4">
+    <select required name="status" class="form-control" ng-change="editing.patient_details.lost_to_follow_up_date = null; editing.patient_details.death_cause = null; editing.patient_details.death_date = null" ng-model="editing.patient_details.status">
+      <option value="" selected>---------</option>
+      <option ng-repeat='i in ["Under Treatment", "Dead", "Lost to Follow-up"] track by $index'>[[ i ]]</option>
```

Comment:
You're sure we want these here?

Reply:
Do you mean the options hard coded into the template?